blueocean-core-js/src/js/components/ResultPageHeader.jsx

```diff
@@ -5,20 +5,25 @@ import {
     HeaderDetails,
     LiveStatusIndicator,
 } from '@jenkins-cd/design-language';
+import i18nTranslator from '../i18n/i18n';
+
+const translate = i18nTranslator('blueocean-web');
```

Comment:
Can we replace this with a prop? The containing component already has a `t` function and global consts complicate testing and reuse.